data/event_bit.py

```diff
@@ -202,6 +202,9 @@
 ENABLE_Y_PARTY_SWITCHING = 0x1ce
 ALWAYS_CLEAR = 0x176 # this event_bit is always clear, used for branching
 
+# bits 0x1e6-0x1ed  Unused, as the SNES versions feature 20 rare item slots rather than 30
+UNLOCKED_PERMA_KT_SKIP = 0x1e6 # custom
```

Comment:
Anyone have an idea where in the WC code would be most proper to clear this event bit when starting the game?
Unsure if it is a real concern or not for this to someone be set somehow on game start.

I tried to add `field.SetEventBit(event_bit.UNLOCKED_PERMA_KT_SKIP)` to start.py but with `-oa 2.0.0 -ob 61.0.0` it would show two "Unlocked Perma KT Skip" dialogs instead of two separate.